markdown/summary.Rmd

```diff
@@ -19,30 +46,29 @@ contact <- data.frame(contact1, contact2, contact3)
+Note that hospital data and National Records of Scotland deaths data are provisional for `r max(basefile$fy)`. This publication will be updated in October `r year(end_date)` with final figures. Figures for the latest year should therefore be treated with caution as there could be an undercount of hospital admissions, particularly for people who died in the last three months of the year.  This is not expected to have a large impact on these published figures when they are revised in subsequent updates.
 
 
 ### Main Points
 
-* In 2018/19, there were 53,168 deaths in Scotland excluding those where an external cause such as unintentional injury was recorded.  For these individuals, 89.2% of their last six months of life was spent either at home or in a community setting, with the remaining 10.8% spent in hospital. This is equivalent to an average of 20 days in hospital in the six months prior to death.
-
-* Over the past nine years, the percentage of time spent at home or in a community setting has gradually increased from 85.3% in 2010/11 to 89.2% in 2018/19.
+* In `r max(basefile$fy)`, there were `r basefile %>% filter(fy == max(fy)) %>% summarise(sum(deaths)) %>% pull(.)` deaths in Scotland excluding those where an external cause such as unintentional injury was recorded.  For these individuals, `r summarise_data(basefile, trend = FALSE) %>% pull(qom)`% of their last six months of life was spent either at home or in a community setting, with the remaining `r 100 - (summarise_data(basefile, trend = FALSE) %>% pull(qom))`% spent in hospital. This is equivalent to an average of 20 days in hospital in the six months prior to death.
 
-**Chart here**
+* Over the past `r as.character(as.english(year(end_date) - year(start_date)))` years, the percentage of time spent at home or in a community setting has gradually increased from `r summarise_data(basefile, trend = TRUE) %>% filter(fy == min(fy)) %>% pull(qom)`% in `r min(basefile$fy)` to `r summarise_data(basefile, trend = FALSE) %>% pull(qom)`% in `r max(basefile$fy)`.
 
-1. Figures for 2018/19 are incomplete and will be revised in October 2019 publication.
+**Percentage of last six months of life spent at home or in a community setting; `r min(basefile$fy)` - `r max(basefile$fy)`**
+![This bar charts shows the QoM measure for each financial year.](figures/figure-1.png) \
 
-* In 2018/19, females aged 85 and older spent a higher percentage of their last six months of life in the community (90.0%) compared to males (87.8%); this equates to females spending, on average an extra four days in the community in their last six months of life.
+* In `r max(basefile$fy)`, females aged 85 and older spent a `r diff_dir` percentage of their last six months of life in the community (`r female`%) compared to males (`r male`%); this equates to females spending, on average an extra `r as.character(day_diff(female, male))` days in the community in their last six months of life.
```

Comment:
The "females" in "this equates to females spending" also needs to be automated. If diff_dir is "lower", this should be males. Can write an if statement in the R chunk so say if diff_dir is higher, this should be "females", otherwise "males"?